merge-sort/merge.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+from timeit import timeit
+
+
+def merge_sort(lst):
+    if len(lst) <= 1:
+        return lst
+    left = []
+    right = []
+    middle = len(lst) // 2
+    for x in lst[middle:]:
+        left.append(x)
```

Comment:
This is not Pythonic.  Just set left to `lst[middle:]` (but also realize that that is in fact the 'right' half of the list)